android_client/ChitChat/app/src/androidTest/java/com/moc/chitchat/ChatListActivityTest.java

```diff
@@ -52,10 +60,11 @@
      * @throws InterruptedException throws in case the Thread.sleep(ms) fails
      */
     @Before
-    public void initialization() throws InterruptedException {
+    public void initialization() throws Exception {
```

Comment:
Could we use the more specific Exceptions rather than just throwing & catching a global Exception please? Not sure how much would need to change for this @aakyol ? But I think it's mostly from the Cryptobox.